qa/src/test/java/io/camunda/migrator/qa/runtime/SkipAndRetryProcessInstancesTest.java

```diff
@@ -49,6 +49,9 @@ class SkipAndRetryProcessInstancesTest extends RuntimeMigrationAbstractTest {
   @Autowired
   private TaskService taskService;
 
+  @Autowired
+  private io.camunda.migrator.HistoryMigrator historyMigrator;
```

Comment:
🔧 Add import instead:
```suggestion
  private HistoryMigrator historyMigrator;
```